browser/src/control/jsdialog/Util.KeyboardGridNavigation.ts

```diff
@@ -0,0 +1,150 @@
+/* -*- js-indent-level: 8 -*- */
+/*
+ * Copyright the Collabora Online contributors.
+ *
+ * SPDX-License-Identifier: MPL-2.0
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * JSDialog.ScrollableBar - helper for creating toolbars with scrolling left/right
```

Comment:
copy-pasted wrong comment cc @Darshan-upadhyay1110

Reply:
https://github.com/CollaboraOnline/online/pull/9940

This PR will fix both the issue